Week1/homework/world-queries.txt

```diff
@@ -1,40 +1,362 @@
+5- What's the top 10 most populated cities?
+ SELECT city.name, city.population FROM city ORDER BY population DESC LIMIT 10;
++-----------------------+------------+
+| name                  | population |
++-----------------------+------------+
+| Mumbai (Bombay)       |   10500000 |
+| Seoul                 |    9981619 |
+| Sï¿½o Paulo           |    9968485 |
+| Shanghai              |    9696300 |
+| Jakarta               |    9604900 |
+| Karachi               |    9269265 |
+| Istanbul              |    8787958 |
+| Ciudad de Mï¿½xico    |    8591309 |
+| Moscow                |    8389200 |
+| New York              |    8008278 |
++-----------------------+------------+
+10 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+6- Select the top 3 countries by population that their names start with the letter ‘P’?
+ SELECT country.name, country.population FROM country WHERE country.name LIKE 'p%' LIMIT 3;
++----------+------------+
+| name     | population |
++----------+------------+
+| Pakistan |  156483000 |
+| Panama   |    2856000 |
+| Pitcairn |         50 |
++----------+------------+
+3 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+7- What is the population in Asia?
+ SELECT Continent, SUM(population) AS 'TOTAL-POPULATION' FROM country WHERE country.continent = 'Asia';
++-----------+------------------+
+| Continent | TOTAL-POPULATION |
++-----------+------------------+
+| Asia      |       3705025700 |
++-----------+------------------+
+1 row in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+8- Which languages are spoken in the Region of ‘South America’?
+SELECT 
+  DISTINCT 
+	countrylanguage.language,
+	countrylanguage.IsOfficial, 
+	country.region 
+  FROM 
+	countrylanguage, 
+	country 
+  WHERE  
+	country.region = 'South America'
+  AND
+	countrylanguage.countrycode = country.code 
+  AND
+	countrylanguage.IsOfficial = 'T' 
+  ORDER BY
+	countrylanguage.language;
++--------------+------------+---------------+
+| language     | IsOfficial | region        |
++--------------+------------+---------------+
+| Aimarï¿½     | T          | South America |
+| English      | T          | South America |
+| Guaranï¿½    | T          | South America |
+| Ketï¿½ua     | T          | South America |
+| Portuguese   | T          | South America |
+| Spanish      | T          | South America |
++--------------+------------+---------------+
+6 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+9- What are the languages spoken on all cities named ‘Barcelona’ (you may need to join 3 tables =) )
+SELECT DISTINCT countrylanguage.language, city.name FROM countrylanguage, city WHERE CITY.NAME = 'Barcelona'AND countrylanguage.IsOfficial = 'T'ORDER BY countrylanguage.language;
```

Comment:
This is a good example of my comment on line 234 about how this form of joining can be confusing, here you forgot to add this condition ` AND countrylanguage.countryCode = city.countryCode `. I would recommend to use joins instead.

Also, you don't need `IsOfficial = 'T'` here because we want to see all the languages.